### ISSUE TITLE
Update wording for links  Hourly and Daily usage on School's home page

### ISSUE THREAD

**Issue by chtowner** [BathHacked/energy-sparks#231]
Links Hourly and Daily usage – wording very confusing. Children think clicking on Daily usage will show them the data for a single Day. Suggest these links are renamed.
Link to current Hourly chart could be renamed - See how your school's energy use varies over a chosen day.
Link to current Daily chart could  be renamed - See how your school's energy use varies over a chosen week.

**Comment by chtowner** [comment]
Not currently clear from home page how you find information for a specific date. School home page should have a direct link to find date specific data. The above rewording of the Hourly link will achieve this.

**Comment by ldodds** [comment]
I've revised the links to make them clearer. The suggested text is too long to fit into the available space, so I've used shorter text. The suggested text will appear if you hover over the link though.

![uploads/6ca74a5a-718f-4d8e-b868-b5cdb8ea2c3a/Screenshot from 2017-08-28 17-39-24.png](https://s3-us-west-2.amazonaws.com/prod.huboard.com/uploads%2F6ca74a5a-718f-4d8e-b868-b5cdb8ea2c3a%2FScreenshot+from+2017-08-28+17-39-24.png)
